src/renderer/components/node/NodeFooter.tsx

```diff
@@ -35,16 +20,7 @@ const NodeFooter = ({
             pr={2}
             w="full"
         >
-            <Center className="nodrag">
-                <Icon
-                    as={isLocked ? LockIcon : UnlockIcon}
-                    color={iconShade}
-                    cursor="pointer"
-                    mb={-1}
-                    mt={-1}
-                    onClick={() => toggleNodeLock(id)}
-                />
-            </Center>
+            <Center className="nodrag" />
```

Comment:
Are those centers on either side necessary?

Reply:
Probably not